docs/tutorials/fitting_procedure.ipynb

```diff
@@ -109,6 +109,8 @@
     "\n",
     "You can also supply an initial guess for the source geometry; if running frank from the terminal, setting `geometry : initial_guess` to `true` in your parameter file will use the `inc`, `pa`, and optionally `dra` and `ddec` in that file as the initial guess for the geometry solver. \n",
     "\n",
+    "**By default, the visibility deprojection in frank scales the total flux to account for the difference between the inclined (observed) brightness and the assumed face-on brightness. This reduces the real component of the observed visibility amplitudes by a factor** $1 / \\cos(i)$ **and increases their weights by a factor** $\\cos(i)^2$. The scaling assumes the emission is optically thick, with an integrated 2D flux of $F = \\cos(i) \\int_r^{r=R}{I(r) 2 \\pi r dr}$. To choose not to rescale the visibilities, set `geometry : rescale_flux` to `false` in your parameter file.  \n",
```

Comment:
Do you mind if we re-phrase this a bit? It's not wrong, but I find it confusing. Is the following ok?

> **By default, the visibility deprojection in frank scales the total flux assuming that the disk is optically thick everywhere. Since an inclined optically thick disk covers less area on the sky, the total brightness should be lower by a factor **$\\cos(i)$**. Internally, this is done by scaling the observed visibility amplitudes by a factor **$1 / \\cos(i)$** and their weights by **$\\cos(i)^2$**. When using frank to compute the visibilites for the model (the 'predicted visibilities') this scaling will automatically be applied to the model visibilties so that their amplitude matches the observed visibilities. This option is the default because it produces an intensity profile that is consistent with one extracted directly from a 2D image (such as an azimuthal average or  major- or minor- axis cut).  If instead you wish to not rescale the visibilities, as appropriate for an optically thin disk,  set `geometry : rescale_flux` to `false` in your parameter file. \n",

I feel that there might be some argument for being even more explicit by including the solution to the radiative transfer equation for a razor thin disk, $I(r) = \\exp(- \\tau(r)/\\cos i)$, and discussing the limits.

Reply:
Great, I changed the text to your version and added a couple sentences about the solution for a flat disk and its limits in 45dff44.